ubuntu/16.04/usr/local/bin/supervisor_start

```diff
@@ -8,6 +8,10 @@ source /usr/local/share/bootstrap/setup.sh
 source /usr/local/share/bootstrap/run_confd.sh
 
 # Initialisation - Runtime installation tasks
+for file in /usr/local/bin/supervisor_custom_start-*; do
+  source ${file}
```

Comment:
I seem to have to `# shellcheck source=/dev/null` to stop this being an issue, as there's no example one it can check